### PR TITLE
Tidies up Mine Drones and Gifts

### PR DESCRIPTION
**Pull request by LordFowl**
Mining drones no longer have all-access. They were never intended to, this was the result of an oversight in my code.

Mining drone's plasma cutter upgrade has been replaced with a KA upgrade and re-added to the mine vendor.

Mining drone's emag module is now a plasma cutter instead of a thermal drill.

The mine vendor has been made tidy(er) again.

Updated Christmas gifts to be more modern.

### CODE REVIEW

File: code/game/objects/items/weapons/gift_wrappaper.dm [hunk 1]
```diff
@@ -265,7 +272,23 @@
...
+		/obj/random/junk,
+		/obj/item/weapon/gun/energy/mousegun,
+		/obj/item/weapon/gun/energy/wand/toy,
+		/obj/item/weapon/mirror,
+		/obj/item/weapon/ore/coal,
+		/obj/item/weapon/ore/coal,
+		/obj/item/weapon/ore/coal,
+		/obj/item/weapon/stamp/clown,
+		/obj/item/organ/heart/skrell,
+		/obj/item/latexballon,
```
Comment (skull132) [inline code/game/objects/items/weapons/gift_wrappaper.dm]: This is going to fail unit test.

File: code/game/objects/items/weapons/gift_wrappaper.dm [hunk 2]
```diff
@@ -312,7 +336,28 @@
...
+		/obj/item/weapon/autochisel,
+		/obj/item/weapon/melee/vaurca/rock,
+		/obj/item/weapon/ore/coal,
+		/obj/item/weapon/ore/coal,
+		/obj/item/weapon/ore/coal,
+		/obj/item/weapon/phone,
+		/obj/item/device/dociler,
+		/obj/item/device/flashlight/maglight,
+		/obj/item/device/megaphone,
+		/obj/item/device/violin,
```
Comment (skull132) [inline code/game/objects/items/weapons/gift_wrappaper.dm]: This too.

**Comment by Sindorman** [comment]
Uhm, you can delete comments in changelod. They are an examples.

**Comment by LordFowl** [comment]
An astute observation. However, there's no point in doing so except for personal satisfaction.

**Comment by chaoko99** [comment]
If I cold make a suggestion; if it isn't too hard, could you make the buy buttons on the point redeption machine use multiple columns, or at least default larger? It's annoyingly difficult to buy multiple of something further down the list.

**Pull request merged**